vizro-ai/src/vizro_ai/__init__.py

```diff
@@ -5,6 +5,6 @@
 
 __all__ = ["VizroAI"]
 
-__version__ = "0.1.1.dev0"
+__version__ = "0.1.1"
```

Comment:
About the preparation of release have we tested using hatch run examples? 
Shall we have guidelines/docs about using it. The assumption of using hatch run example: 1. default model is gpt3.5 from openai  2. openai key is already in running environment

About langchain version bump up we need to confirm it is tested overall.

Reply:
Yes, `hatch run example` works, I would not add it to the docs just yet, because we have no contribution guide (yet), and it's just a simplification for the core developers.

I tested the langchain bump, and it looks fine to me